editors/vscode/src/common.ts

```diff
@@ -148,6 +148,8 @@ export function activate(
             statusBar.hide();
             const cl = client.client;
             if (cl) {
+                // mark as stopped so that we don't detect it as a crash
+                (cl as any)['slint_stopped'] = true;
```

Comment:
Instead of the any cast, I think it would be cleaner to use `Object.defineProperty(cl, "slint_stopped", { value: true });` and then `Object.hasOwnProperty` to check for its existence before (further down) before getting it. But that's just nitpicking, of course I generally agree with this fix :)

In an ideal world, we'd use a [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) for this, but I'm not sure we can easily share that between common.ts and extension.ts? (I just didn't check, maybe it is possible)

Reply:
Not an important tweak for this already merged PR, but you wouldn't need to share the symbols if you use a [symbol key like `Symbol.for("slint-ui/lsp-is-stopped")`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for), which will behave like an atom in elixir/erlang/clojure with a key to a runtime global shared look-up table.